src/sources/kubernetes_logs/mod.rs

```diff
@@ -171,6 +171,10 @@ pub struct Config {
     #[configurable(metadata(docs::type_unit = "bytes"))]
     max_read_bytes: usize,
 
+    /// Instead of balancing read capacity fairly across all watched files, prioritize draining the oldest files before moving on to read data from younger files.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏"younger" -> "newer" ?

Reply:
I think "younger" here is also correct 🤔 I copy/pasted this description from the `file` source.